traffic_portal/test/integration/PageObjects/CDNPage.po.ts

```diff
@@ -139,18 +138,18 @@ export class CDNPage extends BasePage {
 
   public async DeleteCDN(cdn: DeleteCDN): Promise<boolean> {
     let name = cdn.Name + this.randomize;
-    let result = false;
     let basePage = new BasePage();
     await this.btnDelete.click();
     await this.txtConfirmName.sendKeys(name);
     await basePage.ClickDeletePermanently();
-    result = await basePage.GetOutputMessage().then(function (value) {
-      if (cdn.validationMessage == value) {
-        return true;
-      } else {
-        return false;
-      }
-    })
-    return result;
+    return await basePage.GetOutputMessage().then(value => cdn.validationMessage === value);
+  }
+  public async CheckCSV(cdn: CheckCSV): Promise<boolean> {
```

Comment:
If all you need is the name, why not just take a `string`?

Reply:
Fixed to string only.